bin/options.js

```diff
@@ -22,9 +22,10 @@ function getOptions() {
   try {
     var opts = fs.readFileSync(optsPath, 'utf8')
           .trim()
+          .replace(/\\\s+/, '%20')
```

Comment:
- The `+` would turn an escaped space followed by one or multiple unescaped spaces into a single `'%20'`.
- The lack of a `//g` flag would make this regex only transform the first escaped whitespace and leave all others

E.g.:

```javascript
"foo bar\\      \\ baz\\ boo".replace(/\\\s+/, '%20')
"foo bar%20\ baz\ boo"
```